Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -589,21 +589,38 @@ void Renderer::CheckForConfigChanges()
 // Create On-Screen-Messages
 void Renderer::DrawDebugText()
 {
-  if (g_ActiveConfig.bShowFPS)
+  if (g_ActiveConfig.bShowFPS | g_ActiveConfig.bShowVPS | g_ActiveConfig.bShowSpeed)
```

Comment:
```suggestion
  if (g_ActiveConfig.bShowFPS || g_ActiveConfig.bShowVPS || g_ActiveConfig.bShowSpeed)
```

Since a lot of analysis tools and various other forms of tooling will flag using bitwise or with booleans as a potential bug vector